Wurst Client for MC 1.9.X/src/tk/wurst_client/mods/AutoMineMod.java

```diff
@@ -33,9 +33,11 @@ public void onUpdate()
 			|| mc.objectMouseOver.getBlockPos() == null)
 			return;
 		if(Block.getIdFromBlock(mc.theWorld.getBlockState(
-			mc.objectMouseOver.getBlockPos()).getBlock()) != 0)
-			mc.gameSettings.keyBindAttack.pressed = true;
-		else
+			mc.objectMouseOver.getBlockPos()).getBlock()) != 0){
+			if(wurst.mods.autoToolMod.isActive())
+				AutoToolMod.setSlot(mc.objectMouseOver.getBlockPos());
+				mc.gameSettings.keyBindAttack.pressed = true;
```

Comment:
@Alexander01998 I need help in Eclpse, please help me.

1) MCP for 1.9 doesn't exist, what should I do?